xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/export_modal.vm

```diff
@@ -78,23 +90,23 @@
               'showRoot': true,
               'showAttachments': false,
               'showTranslations': false,
-              'readOnly': false
+              'readOnly': false,
+              'filterHiddenDocuments': false
             })
           </div>
         #end
-        <a href="$doc.getURL('export', "format=html&amp;$!escapetool.html($request.getQueryString())")" class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.html')</a>
+        <a href="$doc.getURL('export', "format=html")" class="btn btn-primary"
+           rel="nofollow">$services.localization.render('core.menu.export.html')</a>
         #if($hasAdmin && $services.xar.isXARExportAvailable())
-          #set($exportURL = "format=xar&amp;name=$escapetool.url($doc.fullName)&amp;pages=$escapetool.url($services.model.serialize($doc.documentReference,'default'))")
-          #if ($doc.name == 'WebHome')
-            ## When nested page also export WebPreferences which contain among other things the right of this entity
-            #set($exportURL = "$exportURL&amp;pages=${escapetool.url($services.model.serialize($doc.documentReference.parent,'default'))}.WebPreferences")
-          #end
-          <a href="$doc.getURL('export', "$exportURL&amp;$!docvariant")" class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.xar')</a>
+          #set($exportURL = "format=xar&amp;name=$escapetool.url($doc.fullName)")
```

Comment:
You dropped the ``$!docvariant``. Was it not needed?

Reply:
That's something I actually did not understood in the file: did not find any usage or declaration of this neither in the .vm or in the ExportAction API. If you have any clue I take it.

Reply:
* ``view.vm`` includes ``startpage.vm`` includes ``xwikivars.vm``
* ``view.vm`` includes ``contentview.vm`` includes ``contentheader.vm`` includes ``menus_content.vm`` includes ``export_modal.vm``

``$docvariant`` defined in ``xwikivars.vm`` is available in ``export_modal.vm``.

Reply:
Thanks! But then it's strange because it seems that XAR export in ExportAction does not handle revision or language and from what I see in `xwikivars` docvariant is used only for those two parameters.

Reply:
So should we keep it?